test/ngAnimate/animateCssSpec.js

```diff
@@ -1328,8 +1328,7 @@ describe('ngAnimate $animateCss', function() {
           animator.end();
 
           expect(element.data(ANIMATE_TIMER_KEY)).toBeUndefined();
-          $timeout.flush();
```

Comment:
So it's probably not a breaking change? Should we mention this in the review anyway? (But no under Breaking Changes)